examples/phi3/requirements-quarot.txt

```diff
@@ -1,2 +1,2 @@
 -r requirements.txt
-git+https://github.com/microsoft/TransformerCompression.git@quarot-main
+git+https://github.com/microsoft/TransformerCompression.git@main
```

Comment:
I don't see the quarot code on the main branch. it is still on the `quarot-main` branch. Are they going to merge the branch into main soon?

Reply:
there is no quarot code on main here https://github.com/microsoft/TransformerCompression
I can still see it on https://github.com/microsoft/TransformerCompression/tree/quarot-main

Reply:
The code is not actually released publicly. The reference to github was wrong, not the branch. Corrected.

Reply:
Discussed offline with @devang-ml. We will leave this reference pointing to public repo. When the QuaRot team updates the public repo with the changes from their own private repo, the pass and the example will work. Until then, the pass will be stay non-functional, except for the case where one can and is able to install the QuaRot implementation from the private repo.

Updated the requirements file accordingly.